lib/system.nim

```diff
@@ -3026,5 +3011,8 @@ export widestrs
 import system/io
 export io
 
+import std/quitprocs
+export addQuitProc
```

Comment:
Gah but now ``addQuitProc`` adds more code, always. The previous implementation was better in terms of "you only pay for what you use".

Reply:
You can probably restore this property by placing the list of quit procs as in `.global.` variable inside the `addQuitProc` function. If this function is not used, then its `global` variable can be erased by dead code elimination as well.

Reply:
Please remove this ``import export`` solution, ``system.nim`` is already the biggest module!

Reply:
how about:
* deprecate system.addQuitProc (with a long/infinite deprecation cycle), or soft-deprecate by mentioning quitprocs.addQuitProc
* rename quitprocs.addQuitProc => quitprocs.addExitProc (works with both closure and noconv)
so that both close and noconv will use the sane ordering so long `addExitProc` is used

so no code breaks, but code that wants to use closures and keep sane ordering can use addExitProc

Reply:
I've implemented my suggestion, no more bloat to system